### PR TITLE
Vmtests validator

### PR DESCRIPTION
**Pull request by ehildenb**
Address https://github.com/ethereum/tests/issues/411

The script `test.py` will eventually also do more (eg. convert VMTests to GeneralStateTests). Initial PR in that direction.

@axic

### CODE REVIEW

**Comment by ehildenb** [comment]
Not ready for merge yet, need to write the schema for the VMTests.

File: src/VMTestsFiller/vmArithmeticTest/smod_i256min1Filler.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
         "0f572e5295c57f15886f9b263e2f6d2d6c7b5ec6" : {
             "balance" : "1000000000000000000",
             "nonce" : "0",
-	    "code" : "				2**(256-1)",
+	        "_code" : "2**(256-1)",
```
Comment (axic) [inline src/VMTestsFiller/vmArithmeticTest/smod_i256min1Filler.json:22]: Indentation seems to be off?

**Comment by axic** [comment]
I think you could split the formatting changes into a PR on its own which could be merged without too much hassle.

**Comment by ehildenb** [comment]
@axic do you mean that I should separate all changes to JSON test sources into their own PR, get that merged, then have the second PR which switches over to the python test validator and such?

**Comment by axic** [comment]
@ehildenb yeah sounds good

**Comment by ehildenb** [comment]
Subsumed by #466

**Pull request closed without merging**